tests/integration-tests.bats

```diff
@@ -33,5 +33,11 @@ load ../script/libtest
 @test "Test function: post-nodejs" {
   test_kubeless_function post-nodejs
 }
-
+@test "Test function: pubsub" {
+  skip "XXX(jjo): flaky kafka->zookeeper under minikube, likely from its kube-dns"
```

Comment:
IMO we should land it if everything else is _lgtm_, to have dind->minikube and
cleanup changes in earlier, then work on figuring out the DNS issue and fix
that particular test (specially since I don't think I'd have much time to invest
on debugging this soon).